spec/features/clearance/user_signs_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+require "support/features/clearance_helpers"
+
+feature "User signs in" do
+  scenario "with valid email and password" do
+    create_user "user@example.com", "user", "password"
+    sign_in_with "user@example.com", "password"
+
+    expect_user_to_be_signed_in
+  end
+end
+
+private
+
+def create_user(email, username, password)
```

Comment:
Usually methods like this are generally useful, so we put them in modules in spec/support, then globally include them inside RSpec. That looks like this:

```ruby
# spec/support/authentication.rb
module AuthenticationHelpers
  # you can call this module whatever you want, but suffixing with -Helpers helps prevent name conflicts
  def create_user(email, username, password)
    # code goes here
  end
end

RSpec.configure do |config|
  config.include AuthenticationHelpers # available inside every RSpec test now!
end
```